accelerator/core/serializer/ser_helper.c

```diff
@@ -0,0 +1,364 @@
+/*
+ * Copyright (C) 2020 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+
+#include "ser_helper.h"
+#include "time.h"
+
+#define logger_id ser_logger_id
+
+bool valid_tryte(char* trytes, int len) {
+  const char trinary_alphabet[] = TRINARY_ALPHABET;
+  char elt[2] = {};
+  for (int i = 0; i < len; ++i) {
+    elt[0] = trytes[i];
+    if (!strstr(trinary_alphabet, elt)) {
+      return false;
+    }
+  }
+  return true;
+}
```

Comment:
It can be optimized in the future without calling the `strstr()`.
See [dcurl validate_trytes()](https://github.com/DLTcollab/dcurl/blob/ad933b1a8d75c3d6875e55ebc644a1aa7fa168f3/src/trinary.c#L47)